src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -397,21 +402,51 @@
 
 helps['ad group'] = """
 type: group
-short-summary: Manage Azure Active Directory groups.
+short-summary: Manage groups.
```

Comment:
```suggestion
short-summary: Manage Microsoft Entra ID groups.
```

Reply:
I am trying to align with Azure PowerShell. The PR description has more explanation: https://github.com/Azure/azure-cli/pull/27756#issue-1976129746.